src/json/json_interface.h

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************\
+
+Module: JSON Commandline Interface
+
+Author: Peter Schrammel
+
+\*******************************************************************/
+
+/// \file
+/// JSON Commandline Interface
+
+#ifndef CPROVER_JSON_JSON_INTERFACE_H
+#define CPROVER_JSON_JSON_INTERFACE_H
+
+class cmdlinet;
+
+/// Parses the JSON-formatted command line from stdin
```

Comment:
I think it would be rather important to document the expected JSON syntax! Otherwise you're bound to trial and error...